.github/actions/allure-report/action.yml

```diff
@@ -15,8 +15,16 @@ outputs:
   report-url:
     description: 'Allure report URL'
     value: ${{ steps.generate-report.outputs.report-url }}
+  report-json-url:
+    description: 'Allure report JSON URL'
+    value: ${{ steps.generate-report.outputs.report-json-url }}
 
 runs:
+  env:
+    BUILD_TYPE: ${{ inputs.build_type }}
+    BUCKET: neon-github-public-dev
+    TEST_OUTPUT: /tmp/test_output
```

Comment:
Weird, it looks like GitHub doesn't like such syntax:
```
Error: /nvme/actions-runner/_work/neon/neon/./.github/actions/allure-report/action.yml (Line: 29, Col: 3): Unexpected value 'steps'
Error: System.ArgumentNullException: Value cannot be null. (Parameter 'You are using a composite action but there are no steps provided in /nvme/actions-runner/_work/neon/neon/./.github/actions/allure-report/action.yml.')
   at GitHub.Runner.Worker.ActionManifestManager.ConvertRuns(IExecutionContext executionContext, TemplateContext templateContext, TemplateToken inputsToken, String fileRelativePath, MappingToken outputs)
   at GitHub.Runner.Worker.ActionManifestManager.Load(IExecutionContext executionContext, String manifestFile)
```

Found similar issues in https://github.com/orgs/community/discussions/51280